tsconfig.compodoc.json

```diff
@@ -8,7 +8,7 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "target": "es5",
+    "target": "es6",
```

Comment:
nitpick: Was it deliberate to change compodoc tsconfig target in this PR to es6?